src/sentry/api/bases/organization_events.py

```diff
@@ -53,7 +70,7 @@ def get_snuba_filter(self, request, organization, params=None):
     def get_snuba_params(self, request, organization, check_global_views=True):
         with sentry_sdk.start_span(op="discover.endpoint", description="filter_params"):
             if (
-                len(request.GET.getlist("field"))
+                len(self.get_field_list(organization, request))
                 + len(self.get_equation_list(organization, request))
                 > MAX_FIELDS
             ):
```

Comment:
can this go back to just being

```python
if len(request.GET.getlist("field")) > MAX_FIELDS:
```

though perhaps keeping it this way may be more explicit

Reply:
hmm I considered that, but I think I wanna push for us to use `get_field_list` when available so I'll leave it as is :+1: